.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This file is required to execute tests in Visual Studio Code.

Reply:
I'm using this setting actively and will add the documents on how to use it in development process as well